tests/model/model_import_export.test.ts

```diff
@@ -766,6 +772,21 @@ test("import localized date as string and detect the format", () => {
   expect(getEvaluatedCell(model, "A1")?.formattedValue).toBe("31/12/2020");
 });
 
+test("Week start is automatically added during migration", () => {
+  expect(
+    new Model({
+      version: 17,
```

Comment:
some reason to target 17 and not 19 ?

Reply:
When I started this task, we were on version 17 :D